src/kartikeyDJune20/FirstDifferIndexArray.java

```diff
@@ -0,0 +1,48 @@
+package kartikeyDJune20;
+
+public class FirstDifferIndexArray {
+
+	void firstDifferIndex(int[] input1, int[] input2) {
+
+		if (input1.length == input2.length) {
+
+			boolean flag = true;
+			int index = 0;
+			for (; index < input1.length; index++) {
+
+				if (input1[index] != input2[index]) {
+					flag = false;
+					System.out.println("Values are not matching at index: " + index);
+					int max;
+					int diff = 0;
+					if (input1[index] > input2[index]) {
+						max = input1[index];
+						diff = input1[index] - input2[index];
```

Comment:
What does it min ? FirstDifferIndexArray under this class you have written max index call really Kartikey? 
Copy paste karate time bhi thoda dhyan rakho.

Reply:
i think i have merged two requirements in one class.. will incorporate the changes... but i havent copied it from anyone.. when Maulik was explaining he asked for the diference and that too in +ive So i have created that.. i will make the neccessary changes. Extremly sorry for the inconvenience

Reply:
Changes incorporated... Thanku